storage/data-plane/src/test/java/com/microsoft/azure/storage/TransferManagerTest.groovy

```diff
@@ -738,7 +670,10 @@ class TransferManagerTest extends APISpec {
         data.position(0)
 
         then:
-        FlowableUtil.collectBytesInBuffer(bu.download().blockingGet().body(null)).blockingGet() == data
+        //Due to memory issues, this check only runs on small to medium sized files.
+        if(dataSize < 500){
+            FlowableUtil.collectBytesInBuffer(bu.download().blockingGet().body(null)).blockingGet() == data
```

Comment:
Unlike the next comment, I think we do need to be able to test data sizes larger that 500 bytes. I get that 500MB is maybe too large for the vm, but we should be able to test at least 100MB or so

Reply:
I've replaced that case with 100 Mb scenario and updated the if statement to download files less than or equal to 100 mega bytes.